packages/x-ui-guardian-bundle/src/react/smarts/GuardianSmart.ts

```diff
@@ -158,6 +159,19 @@ export class GuardianSmart<
     await this.load();
   }
 
+  /**
+   * Transforms the user received from retrieveUser().
+   * By default, it only makes _id an ObjectId.
+   * @param user
+   * @returns user
+   */
+  public transformUser(user: TUserType): TUserType {
```

Comment:
in theory what you receive here can be anything, the graphql response might not be the user type, should we make a distinction between "GraphQL Response" vs "Actual User we Store In Guardian" ?